inc/commondbtm.class.php

```diff
@@ -2581,13 +2581,13 @@ function can($ID, $right, array &$input=NULL) {
 
       /* Hook to restrict user right on current item @since 9.2 */
       $this->right = $right;
+      plugin::doHook('can', $this);
```

Comment:
I still wonder if this new hook is OK (see https://github.com/glpi-project/docdev/pull/50)... If not, it should be renamed in the 9.1 branch.